_data/import-public.rb

```diff
@@ -1,20 +1,65 @@
 #! /usr/bin/env ruby
 #
 # Imports project data from the 18F Public Hub API.
-#
-# Author: Mike Bland (michael.bland@gsa.gov)
-# Date:   2014-12-22
 
-require 'open-uri'
-require 'json'
+require "open-uri"
+require "json"
 
 DATA_DIR = File.dirname __FILE__
-TEAM_API_BASE_URL = 'https://team-api.18f.gov/public/api/'
+TEAM_DIR = "_team"
+TEAM_API_BASE_URL = "https://team-api.18f.gov/public/api/"
+TEAM_ENDPOINT = "team"
+MEMBER_PROPERTIES = [
+  "city",
+  "first_name",
+  "full_name",
+  "github",
+  "last_name",
+  "name",
+  "role",
+  "state",
+  "team",
+  "twitter",
+  ].freeze
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.